scripts/download_models.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```

Comment:
Yeah, I think this is a formatting issue. You can/should probably restore upstream's copies.